src/testing/euler_gaussian_bump.cpp

```diff
@@ -70,6 +83,10 @@ ::run_test () const
         pcout << initial_conditions.farfield_conservative[s] << std::endl;
     }
 
+    std::string Error_string;
```

Comment:
avoid using capital letters for variable names

Reply:
Fixed it.